exist-core/src/main/java/org/exist/util/Configuration.java

```diff
@@ -887,558 +929,472 @@ private void configureBackend( final Optional<Path> dbHome, Element con ) throws
-                LOG.warn( e );
+            } catch (final NumberFormatException e) {
+                LOG.warn(e);
             }
         }
 
-        final String maxOutput = getConfigAttributeValue( watchDog, "output-size-limit" );
+        final String maxOutput = getConfigAttributeValue(watchDog, "output-size-limit");
 
-        if( maxOutput != null ) {
+        if (maxOutput != null) {
 
             try {
-                config.put( XQueryWatchDog.PROPERTY_OUTPUT_SIZE_LIMIT, Integer.valueOf(maxOutput) );
+                config.put(XQueryWatchDog.PROPERTY_OUTPUT_SIZE_LIMIT, Integer.valueOf(maxOutput));
                 LOG.debug(XQueryWatchDog.PROPERTY_OUTPUT_SIZE_LIMIT + ": {}", config.get(XQueryWatchDog.PROPERTY_OUTPUT_SIZE_LIMIT));
-            }
-            catch( final NumberFormatException e ) {
-                LOG.warn( e );
+            } catch (final NumberFormatException e) {
+                LOG.warn(e);
             }
         }
     }
 
-
     /**
      * DOCUMENT ME!
      *
-     * @param  queryPool
+     * @param queryPool element with queryPool settings
      */
-    private void configureXQueryPool( Element queryPool )
-    {
-        final String maxStackSize = getConfigAttributeValue( queryPool, XQueryPool.MAX_STACK_SIZE_ATTRIBUTE );
+    private void configureXQueryPool(final Element queryPool) {
+        final String maxStackSize = getConfigAttributeValue(queryPool, XQueryPool.MAX_STACK_SIZE_ATTRIBUTE);
 
-        if( maxStackSize != null ) {
+        if (maxStackSize != null) {
 
             try {
-                config.put( XQueryPool.PROPERTY_MAX_STACK_SIZE, Integer.valueOf(maxStackSize) );
+                config.put(XQueryPool.PROPERTY_MAX_STACK_SIZE, Integer.valueOf(maxStackSize));
                 LOG.debug(XQueryPool.PROPERTY_MAX_STACK_SIZE + ": {}", config.get(XQueryPool.PROPERTY_MAX_STACK_SIZE));
-            }
-            catch( final NumberFormatException e ) {
-                LOG.warn( e );
+            } catch (final NumberFormatException e) {
+                LOG.warn(e);
             }
         }
 
-        final String maxPoolSize = getConfigAttributeValue( queryPool, XQueryPool.POOL_SIZE_ATTTRIBUTE );
+        final String maxPoolSize = getConfigAttributeValue(queryPool, XQueryPool.POOL_SIZE_ATTTRIBUTE);
 
-        if( maxPoolSize != null ) {
+        if (maxPoolSize != null) {
 
             try {
-                config.put( XQueryPool.PROPERTY_POOL_SIZE, Integer.valueOf(maxPoolSize) );
+                config.put(XQueryPool.PROPERTY_POOL_SIZE, Integer.valueOf(maxPoolSize));
                 LOG.debug(XQueryPool.PROPERTY_POOL_SIZE + ": {}", config.get(XQueryPool.PROPERTY_POOL_SIZE));
-            }
-            catch( final NumberFormatException e ) {
-                LOG.warn( e );
+            } catch (final NumberFormatException e) {
+                LOG.warn(e);
             }
         }
     }
-    
-    public static class StartupTriggerConfig {
```

Comment:
Removed..... can't find replacement?

Reply:
The class `StartupTriggerConfig` was replaced with/ converted to a record.

Reply:
in deae76a7cee3a4654c2cd577db3fd5c6c7573094

Reply:
@dizzzz Please resolve if you do not have further questions